scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1760,10 +1762,22 @@ class Router(formatOps: FormatOps) {
       case FormatToken(c: T.Comment, _, _) =>
         Seq(Split(getMod(formatToken), 0))
 
-      case FormatToken(_: T.KwImplicit, _, _)
+      case FormatToken(_: T.KwImplicit | T.Ident("using"), _, _)
           if !style.binPack.unsafeDefnSite &&
-            !style.verticalMultiline.atDefnSite =>
-        opensImplicitParamList(prevNonComment(prev(formatToken))).fold {
+            !style.verticalMultiline.atDefnSite && isRightImplicitOrUsingSoftKw(
+              prev(formatToken)
+            ) =>
+        val argsOrParamsOpt = formatToken.meta.leftOwner match {
+          // for the using argument list
+          case _: ApplyUsing =>
+            Some {
+              val (_, args) = getApplyArgs(formatToken, false)
```

Comment:
why not just `getApplyArgs(...)._2`?

Reply:
That was my suggestion. I really dislike `_2` etc. but I think maybe it's way too complex now.

Reply:
we can also create a new case class and have `getApplyArgs` return it, instead of the tuple.